test/sql/resolve_connection_filter.sql

```diff
@@ -121,6 +122,86 @@ begin;
     select graphql.resolve($${accountCollection(filter: {phone: {is: null}}) { edges { node { id } } }}$$);
     rollback to savepoint a;
 
+    -- cs - array column contains the input scalar
+    select jsonb_pretty(
+        graphql.resolve($$
+            {
+              accountCollection(filter: {tags: {cs: "customer"}}) {
+                edges {
+                  node {
+                    id
+                  }
+                }
+              }
+            }
+        $$)
+    );
+    rollback to savepoint a;
+
+    -- cd - array column is contained by input scalar (aka, the only value in the array column is the input scalar)
+    select jsonb_pretty(
+        graphql.resolve($$
+            {
+              accountCollection(filter: {tags: {cd: "customer"}}) {
+                edges {
+                  node {
+                    id
+                  }
+                }
+              }
+            }
+        $$)
+    );
+    rollback to savepoint a;
+
+    -- cs - array column contains the input array
+    select jsonb_pretty(
+        graphql.resolve($$
+            {
+              accountCollection(filter: {tags: {cs: ["customer", "priority"]}}) {
+                edges {
+                  node {
+                    id
+                  }
+                }
+              }
+            }
+        $$)
+    );
+    rollback to savepoint a;
+
+    -- cd - array column is contained by input array
+    select jsonb_pretty(
+        graphql.resolve($$
+            {
+              accountCollection(filter: {tags: {cd: ["customer", "priority"]}}) {
+                edges {
+                  node {
+                    id
+                  }
+                }
+              }
+            }
+        $$)
+    );
+    rollback to savepoint a;
+
+    -- ov - array column overlaps with input array
+    select jsonb_pretty(
+        graphql.resolve($$
+            {
+              accountCollection(filter: {tags: {cd: ["customer", "priority"]}}) {
```

Comment:
`ov` filter here?

Reply:
Yes, nice catch!! It was a copy-paste casualty 😆